library/exampleComponents/Icons/facebookIcon.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
Let's call this file facebook.js instead of facebookIcon.js and the same for all icons.
Also, there are warnings in browser console caused by properties like stroke-width passed to icon element, all these props needs to be converted to camelCase